mets_mods2tei/api/mets.py

```diff
@@ -331,7 +331,8 @@ def __spur(self):
                     self.img_map[div.get_ID()] = default_map[fptr.get_FILEID()]
 
         # struct links
-        for sm_link in self.tree.xpath("//mets:structLink", namespaces=ns)[0].iterchildren():
+        structlinks = self.tree.xpath("//mets:structLink/*", namespaces=ns)
```

Comment:
Is there more than readability to this proposed change?

Reply:
Yes – as the changelog description says: there might be no `structLink` element at all, which used to crash, but we only want an empty loop.

Reply:
Forget this comment. The commit-by-commit reading is not optimal.